realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -226,9 +228,8 @@ protected void removeHandler() {
      * the last transaction was committed.
      *
      * @param destination file to save the Realm to.
-     * @throws java.io.IOException if any write operation fails.
      */
-    public void writeCopyTo(File destination) throws java.io.IOException {
+    public void writeCopyTo(File destination) {
```

Comment:
Ideally, I would probably like to keep the IOException for problems with the destination + add a comment about IllegalArgumentException being thrown if invalid Realm file.

But I am not sure if the ObjectStore exposes that as two different exceptions, if not just adding a comment about IllegalArgumentException is probably the next best thing